modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -123,3 +123,65 @@ endif::[]
 
 {centos} clients are registered in the same way as all other clients.
 For more information, see xref:client-configuration:registration-overview.adoc[].
+
+
+
+== Manage Errata
+
+When you update {centos} clients, the packages do not include metadata about the updates.
+You can use a third-party errata service to obtain this information.
+
+ifeval::[{suma-content} == true]
+
+[IMPORTANT]
+====
+The third-party errata service described here is provided and maintained by the community.
+It is not supported by {suse}.
+====
+
+endif::[]
+
+.Procedure: Installing an Errata Service
+
+. From the command prompt, as root, add the ``sle-module-development-tools`` module:
++
+----
+SUSEConnect --product sle-module-development-tools/15.1/x86_64
+----
+. Install the errata service:
++
+----
+zypper in  perl-Text-Unidecode
+----
+. Create a file for your errata script:
++
+----
+touch /usr/local/bin/cent-errata.sh
+----
+. Edit the new file to include this script, editing the repository details as required.
+This script fetches the errata details from an external errata service, unpacks it, and publishes the details:
++
+----
+#!/bin/bash
+mkdir -p /usr/local/centos
+cd /usr/local/centos
+rm *.xml
+wget -c http://cefs.steve-meier.de/errata.latest.xml
+#wget -c https://www.redhat.com/security/data/oval/com.redhat.rhsa-all.xml
+wget -c https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml
+wget -c http://cefs.steve-meier.de/errata-import.tar
+tar xvf errata-import.tar
+chmod +x /usr/local/centos/errata-import.pl
+export SPACEWALK_USER='admin';export SPACEWALK_PASS='<password>'
```

Comment:
'admin' needs to be replaced with the real name as well like the password.
Not sure if we should write `<user>` .